pkg/plugins/autodiscovery/cargo/dependencies.go

```diff
@@ -0,0 +1,207 @@
+package cargo
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"sort"
+	"text/template"
+
+	"github.com/updatecli/updatecli/pkg/plugins/utils/cargo"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// ValidFiles specifies accepted Cargo files.
+	ValidFiles [1]string = [1]string{"Cargo.toml"}
+)
+
+type crateDependency struct {
+	Name     string
+	Registry string
+	Version  string
+	Inlined  bool
+}
+
+type crateMetadata struct {
+	Name            string
+	Dependencies    []crateDependency
+	DevDependencies []crateDependency
+}
+
+func (c Cargo) generateManifest(crateName string, dependency crateDependency, relativeFile string, foundFile string, dependencyType string) (bytes.Buffer, error) {
+	manifest := bytes.Buffer{}
+	tmpl, err := template.New("manifest").Parse(dependencyManifest)
+	if err != nil {
+		logrus.Debugln(err)
+		return manifest, err
+	}
+	var existingSourceKey string
+	if dependency.Inlined {
+		existingSourceKey = fmt.Sprintf("%s.%s", dependencyType, dependency.Name)
+	} else {
+		existingSourceKey = fmt.Sprintf("%s.%s.version", dependencyType, dependency.Name)
+	}
+	var ConditionQuery string
+	if dependency.Inlined {
+		ConditionQuery = fmt.Sprintf("%s.(?:-=%s)", dependencyType, dependency.Name)
+	} else {
+		ConditionQuery = fmt.Sprintf("%s.(?:-=%s).version", dependencyType, dependency.Name)
+	}
+	var TargetKey string
+	if dependency.Inlined {
+		TargetKey = fmt.Sprintf("%s.%s", dependencyType, dependency.Name)
+	} else {
+		TargetKey = fmt.Sprintf("%s.%s.version", dependencyType, dependency.Name)
+	}
+	var Registry cargo.Registry
+	if dependency.Registry != "" {
+		if registry, found := c.spec.Registries[dependency.Registry]; found {
+			Registry = registry
+		}
+	}
+	params := struct {
+		ManifestName               string
+		CrateName                  string
+		DependencyName             string
+		SourceID                   string
+		SourceName                 string
+		SourceVersionFilterKind    string
+		SourceVersionFilterPattern string
+		ExistingSourceID           string
+		ExistingSourceName         string
+		ExistingSourceKey          string
+		ConditionID                string
+		ConditionQuery             string
+		File                       string
+		TargetID                   string
+		TargetFile                 string
+		TargetKey                  string
+		ScmID                      string
+		WithRegistry               bool
+		RegistrySCMID              string
+		RegistryRootDir            string
+		RegistryURL                string
+		RegistryAuthToken          string
+		RegistryHeaderFormat       string
+	}{
+		ManifestName:               fmt.Sprintf("Bump %s %q for \"%s\" crate", dependencyType, dependency.Name, crateName),
```

Comment:
```suggestion
		ManifestName:               fmt.Sprintf("Bump %s %q for %q crate", dependencyType, dependency.Name, crateName),
```

%q surround the string value with quotes